sigs.yaml

```diff
@@ -129,6 +129,39 @@ sigs:
           - github: xpivarc
             name: "Luboslav Pivarc"
             company: Red Hat
+    - dir: sig-ci
+      name: KubeVirt CI Operations Group
+      mission_statement: |
+        Ensure that the CI infrastructure is operational, that CI jobs are executed in a timely manner and PRs of any of the onboarded projects are not blocked.
+      leadership:
+        chairs:
+        - github: brianmcarey
+          name: "Brian Carey"
+          company: Red Hat
+        - github: dhiller
+          name: "Daniel Hiller"
+          company: Red Hat
+      meetings:
+      - description: KubeVirt CI Taskforce Sync meeting
```

Comment:
@aburdenthehand qq wrt to the Google Calendar: can I create the Google Calendar agenda item myself? Or do you need to do this?

Reply:
IIUC it needs to be done using the KubeVirt account for it to show up on the shared calendar and mirrored on the CNCF calendar.